conftest.py

```diff
@@ -11,10 +11,6 @@ def pytest_configure(config):
     # Note: We could manually register/configure INSTALLED_APPS by traversing our entry points
     # or package directories, but this is easier assuming Sentry doesn't change APIs.
     # Note: Order of operations matters here.
-    from sentry.runner.importer import install_plugin_apps
-
-    install_plugin_apps("sentry.apps", settings)
```

Comment:
Why did this get removed? Do slack/pd not need to be added to INSTALLED_APPS by sentry's importer?

Reply:
Actually yeah, seems like we should keep this since there are still a couple plugins in `sentry.apps`

Reply:
@wedamija @joshuarli Hmmm? The only ones in that list were Jira, Jira AC, and SessionStack. All of those are now in sentry-core which has the entrypoint of `sentry.new_apps`.

Reply:
@scefali Ah you're right, I understand now.